js/wrapper.js

```diff
@@ -22,18 +31,22 @@ let x_pos = 0, y_pos = 0; // Stores x & y coordinates of the mouse pointer
 let x_elem = 0, y_elem = 0; // Stores top, left values (edge) of the element
 
 // Resize
-let startXResize, startYResize, startWidthResize, startHeightResize;
+let startWidthResize, startHeightResize;
 
 chrome.runtime.onMessage.addListener(function (message) {
     if (message.type) {
         if (message.type === createType) {
-            startVideo(message.text, message.isHidden)
+            playerInfos = playerInfos || message.playerInfos;
```

Comment:
Just want to know why we don't update with the message content everytime if available
Shouldn't you flip the OR ?
Appart from that, good for me, GJ @nugetchar :)

Reply:
@ImFlog  That's because the message.playerInfos might be undefined.
In background.js, we also have a ```playerInfos``` variable. This variable is set only when we update the position and/or the dimensions of the player (message sent from ```wrapper.js``` to ```backrgound.js```). Otherwise, we must reaffect the ```playerInfos``` variable of the ```wrapper.js``` file in order to avoid affecting an ```undefined``` value.

Reply:
And now that I'm reading this, I realize that you might be right: we should flip the OR condition.